### PR TITLE
fix:Veracode CVEs fix

### PR DESCRIPTION
**Pull request by shijinrajbosch**
## Description

Veracode CVEs fix in jetty-http jar.

- CVE-2023-36478 | CWE-190
Denial Of Service (DoS): org.eclipse.jetty is vulnerable to Denial Of Service (DoS). The vulnerability arises from the library's failure to appropriately limit the size in HPACK header values. This allows an attacker to repeatedly send maliciously crafted HTTP messages, leading to an integer overflow and ultimately causing an application crash through the `checkSize` function in `MetaDataBuilder.java`.

- CVE-2023-40167| CWE-130
HTTP Request Smuggling: Jetty is vulnerable to HTTP Request Smuggling. The vulnerability is due to accepting `+` character proceeding the content-length in the request. This vulnerability can be exploited by the attacker to possibly conduct request smuggling attacks.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.3.19
```
Comment (tunacicek) [inline CHANGELOG.md:7]: Please remove this version and add changes to the 0.3.18.

Comment (shijinrajbosch) [reply CHANGELOG.md]: > Please remove this version and add changes to the 0.3.18.

Thanks for the review comments @tunacicek . I have made the changes.

**Review by agg3fe — APPROVED** [review]
LGTM

**Review by tunacicek — APPROVED** [review]
LGTM

**Pull request merged**